### PR TITLE
add objects and selectors for comms screen page

### PR DESCRIPTION
**Pull request by ditmirDVSA**
MES-7022

### CODE REVIEW

File: src/support/globalelements/globalelements.json [hunk 1]
```diff
@@ -118,6 +122,24 @@
...
+            "new-email-error-msg":"#communication-new-email-validation-text",
+            "select-bypost-radio-btn":"input[id='postalAddress']",
+            "by-post-label":"label[for='postalAddress']",
+            "address-container":".display-address",
+            "address-line-1":"#addressLine1",
+            "address-line-2":"#addressLine2",
+            "address-line-3":"#addressLine3",
+            "address-postcode":"#postcode"
+        }
+    ],
```
Comment (turnerc1) [inline src/support/globalelements/globalelements.json:142]: There is also a "by email (provided at booking time)" field which is not mentioned here. Are you able to do this field as well please?

Comment (turnerc1) [reply src/support/globalelements/globalelements.json:142]: AS well as the back button, screen title, end test button? Are you able to do these as well.

**Review by turnerc1 — CHANGES_REQUESTED** [review]

File: src/support/globalelements/globalelements.json (hunk 1 above)
Comment (ditmirDVSA) [reply src/support/globalelements/globalelements.json:142]: The by email is already specified here             "select-email-radio-btn":"input[id='newEmail']",
 
The other buttons for end test and back are specified too in the globalelements, I will however move these buttons into des-general as they apply to many pages

Comment (turnerc1) [reply src/support/globalelements/globalelements.json:143]: You have the "New Email" but not the existing email. CHeck the miro board for this page.

Comment (ditmirDVSA) [reply src/support/globalelements/globalelements.json:143]: That option is not built 

Screenshot 2021-08-09 at 11 27 32

Comment (turnerc1) [reply src/support/globalelements/globalelements.json:143]: That option is build check other tests

Comment (ditmirDVSA) [reply src/support/globalelements/globalelements.json:143]: which tests ? This is the app I've loaded , doesn't appear it is on there

**Review by turnerc1 — APPROVED** [review]

**Pull request merged**